ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -496,6 +496,62 @@
 | L3 Gateway Inter-domain | True |
 {%         endif %}
 {%     endif %}
+{%     if router_bgp.address_family_vpn_ipv4 is arista.avd.defined %}
+
+### Router BGP VPN-IPv4 Address Family
+{%         if router_bgp.address_family_vpn_ipv4.route.import_match_failure_action is arista.avd.defined('discard') %}
+
+- VPN import pruning is __enabled__
+{%         endif %}
+{%         if router_bgp.address_family_vpn_ipv4.neighbors is arista.avd.defined %}
+
+#### VPN-IPv4 Neighbors
+
+| Neighbor | Activate |
+| -------- | -------- |
+{%             for neighbor in router_bgp.address_family_vpn_ipv4.neighbors | arista.avd.convert_dicts('ip_address') | arista.avd.natural_sort('ip_address') %}
+| {{ neighbor.ip_address }} | {{ neighbor.activate | arista.avd.default(false) }} |
+{%             endfor %}
+{%         endif %}
+{%         if router_bgp.address_family_vpn_ipv4.peer_groups is arista.avd.defined %}
+
+#### VPN-IPv4 Peer Groups
+
+| Peer Group | Activate |
+| ---------- | -------- |
+{%             for peer_group in router_bgp.address_family_vpn_ipv4.peer_groups | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
+| {{ peer_group.name }} | {{ peer_group.activate | arista.avd.default(false) }} |
+{%             endfor %}
+{%         endif %}
+{%     endif %}
+{%     if router_bgp.address_family_vpn_ipv6 is arista.avd.defined %}
+
+### Router BGP VPN-IPv6 Address Family
+{%         if router_bgp.address_family_vpn_ipv6.route.import_match_failure_action is arista.avd.defined('discard') %}
+
+- VPN import pruning is __enabled__
```

Comment:
Do we want a statement here if VPN import pruning is disabled?  Maybe something like 'VPN import pruning is disabled by default' ?